src/runtime/runtime_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build darwin linux,!baremetal freebsd,!baremetal
+// +build (darwin linux,!baremetal freebsd,!baremetal),!rpi3
```

Comment:
The board is marked as `baremetal`, which will make this unnecessary.

Reply:
Yep, fixed.